packages/page-mixer/src/components/deposit/DepositConsole.tsx

```diff
@@ -1,36 +1,25 @@
-import { BalanceInputValue, eliminateGap } from '@webb-dapp/react-components';
+import { BalanceInputValue } from '@webb-dapp/react-components';
 import AmountInput from '@webb-dapp/react-components/AmountInput/AmountInput';
 import { TokenInput } from '@webb-dapp/react-components/TokenInput';
-import { useApi, useBalance, useBalanceValidator, useConstants } from '@webb-dapp/react-hooks';
+import { useApi, useConstants, useMixer } from '@webb-dapp/react-hooks';
```

Comment:
use `useMixerProvider` instead of `useMixer` which will point to the value from the context,
`useMixer` will return a new mixer ignoring the context

The context for the mixer doesn't contain the `mixerInfos`

```suggestion
import { useApi, useConstants, useMixerProvider , useMixerInfos} from '@webb-dapp/react-hooks';
```